### PR TITLE
Fixed wrong file path in README

### PR DESCRIPTION
**Pull request by mmmavis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Activate the virtual environment:
 Install all dependencies into the virtual environment:
 
 ```bash
-pip install -r requirements.txt
+pip install -r ../requirements.txt
```
Comment (gideonthomas) [inline README.md]: probably add a Windows path as well

Comment (mmmavis) [reply README.md]: not sure how. can you patch this PR?

**Comment by gideonthomas** [comment]
@mmmavis, I **think** I got the Windows path right lol

**Comment by mmmavis** [comment]
@gideonthomas thanks!

**Comment by cadecairos** [comment]
heh, just saw that the windows path was a nit @gideonthomas had. It's little known that the type of slash you use doesn't matter in windows command prompts.

**Comment by gvn** [comment]
@mmmavis - Needs a rebase. 🎉

**Comment by mmmavis** [comment]
I just did a rebase and realized @gvn's [PR](https://github.com/mozilla/foundation.mozilla.org/pull/907) has already fixed the issue this PR was trying to address. Closing this one now.

**Pull request closed without merging**